packages/api/src/server/sot/rover/rover.controller.ts

```diff
@@ -18,12 +18,11 @@ export class RoverController {
     return this.searchService.getRoverUserDetails(username);
   }
 
-  @Get('/rover/group/:groupname')
+  @Get('/rover/group/:groupName')
   @ApiCreatedResponse({ status: 201, description: 'List for the users in Rover Group.', type: Subject, isArray: true })
   @ApiOperation({ description: 'Search the user details from Rover Group.' })
-  async getRoverGroupDetails(@Param('groupname') groupname: string, @Query('list') list: string,): Promise<Subject[]> {
-    if (list === 'group') return this.searchService.getRoverGroupList(groupname);
-    return this.searchService.getRoverGroupDetails(groupname);
+  async getRoverGroupDetails(@Param('groupName') groupName: string, @Query('list') list: string): Promise<Subject[]> {
+    if (list === 'group') return this.searchService.getRoverGroupList(groupName);
```

Comment:
'goup' seems to be a string, and list === 'group' is confusing. Can you re-write this?

Reply:
as of now there is a single Attribute, in future we can have a comma separated query in the list, so 'list' was added. 

what about 'searchBy === group' !?

Reply:
can you please change   to `searchByCriteria === 'group'`

Reply:
Hi @arkaprovob , I've made the changes, please check. this is also noted, from the next time onwards, we'll follow this practice.